Source/WebKit/WebProcess/gtk/WebProcessMainGtk.cpp

```diff
@@ -60,6 +61,7 @@ class WebProcessMainGtk final: public AuxiliaryProcessMainBase<WebProcess> {
 public:
     bool platformInitialize() override
     {
+        WTF::StackBounds::setBottomOfMainThreadMain(__builtin_frame_address(0));
```

Comment:
I have no idea if it would make sense or not, but maybe this should be called in `WebProcessMain()` instead?